test/tests.js

```diff
@@ -244,10 +244,26 @@ describe('Braze Forwarder', function() {
             };
         },
         reportService = new ReportingService();
-
+        mParticle.Identity = {
+            getCurrentUser: function () {
+                return {
+                    getMPID: function () {
+                        return 'MPID123';
+                    },
+                    getUserIdentities: function () {
```

Comment:
is `getUserIdentities` a necessary addition?

Reply:
Currently, it is necessary, yes. The getUserIdentities function is needed in the onUserIdentified call which sets the Braze identity. In retrospect, I'm calling the onUserIdentified function which in turn calls the Braze changeUser function. Seems like I could skip the secondary function and call the Braze function directly - would it make sense to make that change to eliminate the getUserIdentities and the extra processing?